recipes/gs-tama/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "1.0" %}
+
+package:
+  name: gs-tama
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/sguizard/gs-tama/archive/refs/tags/1.0.tar.gz
+  sha256: 11e7e1aadc5a9a5f8eb4b45aedc8217d937e15fa59b2ad7f1904c32a08c76062
+
+requirements:
+  run:
+    - python=2.7.18
+    - biopython=1.76
+    - pysam=0.16.0.1
+    - bedtools=2.30.0
+    - blast=2.11.0
+    - samtools=1.12
```

Comment:
Are these exact versions required, or might newer versions work as well?

Reply:
It should work fine with newer versions. 
I just set up those because it the versions with I tested the package.

Reply:
Am I correct that this is compatible with python3?

Reply:
It has been developed for python2. 
I didn't tested the compatibility with python3. 
Maybe I'll able to check that this coming weeks.